maps/tether/tether_areas.dm

```diff
@@ -24,7 +24,7 @@
 // Elevator areas.
 /area/turbolift
 	delay_time = 2 SECONDS
-	forced_ambience = list('sound/music/elevator.ogg')
+	forced_ambience = list('modular_citadel/sound/music/elevator.ogg')
```

Comment:
`//CITADEL CHANGE - changes forced_ambience here to modular_citadel/sound/music/elevator.ogg`